pkg/news/news_test.go

```diff
@@ -121,6 +121,9 @@ func TestPrintNewsFeed(t *testing.T) {
 		{name: "latest-quiet", args: args{bottomUp: true, cutOffDate: lastNewsTime, all: false, quiet: true}, wantErr: false},
 		{name: "latest-quiet-topdown", args: args{bottomUp: false, cutOffDate: lastNewsTime, all: false, quiet: true}, wantErr: false},
 	}
+	currentTZ := os.Getenv("TZ")
+	defer os.Setenv("TZ", currentTZ)
+	os.Setenv("TZ", "UTC")
```

Comment:
I think you can use `t.Setenv("TZ", "UTC")` instead which will automatically reset the env at the end of the test

Reply:
Done.